src/overlays/actors/ovl_En_Ganon_Mant/z_en_ganon_mant.c

```diff
@@ -21,19 +27,444 @@ const ActorInit En_Ganon_Mant_InitVars = {
     (ActorFunc)EnGanonMant_Update,
     (ActorFunc)EnGanonMant_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Ganon_Mant/EnGanonMant_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Ganon_Mant/EnGanonMant_Destroy.s")
+static s16 sTearSizesMedium[] = {
+    0, 0, 0, 1, 1, 1, 1, 1, 0, 0,
+};
+
+static s16 sTearSizesLarge[] = {
+    0, 0, 0, 0, 1, 1, 2, 2, 2, 1, 1, 0, 0, 0, 0,
+};
+
+static s16 sTearSizesSmall[] = {
+    0, 0, 0, 0, 0, 0, 0,
+};
+
+typedef struct {
+    s16* tearAreaSizes;
+    s16 count;
+} TearShape; // size = 0x8
+
+/**
+ * The arrays pointed to by this table describe how many pixels should
+ * be removed from the cloak texture in a single pass
+ */
+static TearShape sTearShapes[] = {
+    { sTearSizesMedium, ARRAY_COUNT(sTearSizesMedium) },
+    { sTearSizesMedium, ARRAY_COUNT(sTearSizesMedium) },
+    { sTearSizesLarge, ARRAY_COUNT(sTearSizesLarge) },
+    { sTearSizesSmall, ARRAY_COUNT(sTearSizesSmall) },
+};
+
+/// How much each joint is affected by backwards/forwards swaying motion
+static f32 sBackSwayCoefficients[GANON_MANT_NUM_JOINTS] = {
+    0.0f, 1.0f, 0.5f, 0.25f, 0.1f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f,
+};
+
+static f32 D_80A24DB4[] = {
+    0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f,
+};
```

Comment:
is this unreferenced or is my ctrl f being dumb on browser lmao
if yes, how do we know these are floats?

Reply:
The unnamed arrays are unreferenced. I assume they are floats as the surrounding arrays are floats.